plugins/com.google.cloud.tools.eclipse.login/META-INF/MANIFEST.MF

```diff
@@ -15,32 +15,18 @@ Require-Bundle: com.google.gson;bundle-version="[2.7,3.0)",
  org.eclipse.ui,
  org.mortbay.jetty.server,
  org.mortbay.jetty.util
-Bundle-Classpath: lib/google-api-client-1.22.0.jar,
- lib/google-api-services-oauth2-v2-rev114-1.22.0.jar,
- lib/google-http-client-1.22.0.jar,
- lib/google-http-client-jackson-1.22.0.jar,
- lib/google-oauth-client-1.22.0.jar,
- lib/google-oauth-client-java6-1.22.0.jar,
- lib/google-oauth-client-jetty-1.22.0.jar,
- lib/plugins-login-common-0.1.1-SNAPSHOT.jar,
- .
-Export-Package: com.google.api.client.auth.oauth2;x-friends:="com.google.cloud.tools.eclipse.appengine.deploy,com.google.cloud.tools.eclipse.appengine.deploy.ui,com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.googleapis;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.googleapis.json;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.googleapis.services;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.googleapis.services.json;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.googleapis.testing.json;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.http;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.http.javanet;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.json;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.json.jackson;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.api.client.util;x-friends:="com.google.cloud.tools.eclipse.projectselector",
- com.google.cloud.tools.eclipse.login,
- com.google.cloud.tools.eclipse.login.ui,
- com.google.cloud.tools.login;version="0.1.1"
-Import-Package: com.google.cloud.tools.eclipse.ui.util,
+Bundle-Classpath: .
+Export-Package: com.google.cloud.tools.eclipse.login,
+ com.google.cloud.tools.eclipse.login.ui
+Import-Package: com.google.api.client.auth.oauth2,
+ com.google.api.client.extensions.jetty.auth.oauth2,
+ com.google.api.client.googleapis.auth.oauth2,
+ com.google.api.client.http,
+ com.google.api.client.json,
+ com.google.cloud.tools.eclipse.ui.util,
  com.google.cloud.tools.eclipse.usagetracker,
  com.google.cloud.tools.eclipse.util,
+ com.google.cloud.tools.login;version="0.1.1",
  javax.servlet.http;version="3.1.0",
```

Comment:
Seems like we can remove this too. At least I had no problem logging in and out.